README.markdown

```diff
@@ -1,4 +1,7 @@
-JSZip
+JSZip [![Build Status](https://api.travis-ci.org/Stuk/jszip.svg?branch=master)](http://travis-ci.org/Stuk/jszip) [![Code Climate](https://codeclimate.com/github/Stuk/jszip/badges/gpa.svg)](https://codeclimate.com/github/Stuk/jszip)
```

Comment:
let's remove the generated files :)

https://docs.codeclimate.com/docs/excluding-files-and-folders